guides/MAC_SETUP.md

```diff
@@ -6,13 +6,78 @@ This document is part of the [Getting Started](https://github.com/ONSdigital/dp/
 Install Applications
 ----
 
-1. Install your preferred browser - the team use a combination of [Chrome](https://www.google.co.uk/chrome) and [Firefox](https://www.mozilla.org/en-GB/firefox) mostly, but the choice is yours.
-
-2. Install [Slack](https://slack.com/intl/en-gb/downloads/mac?geocode=en-gb) - used for much of the communication (text, voice, video, screen-sharing) within the team/department (and beyond).
-
-3. (Optional) Install [iTerm2](https://www.iterm2.com/downloads.html) or your terminal of choice.
-
-4. (Strongly recommended) Install [homebrew](https://brew.sh/) - a package manager - useful for installing (and keeping up-to-date) apps on macOS.
+1. (Optional) Install [iTerm2](https://www.iterm2.com/downloads.html) (or your terminal of choice).
```

Comment:
I'd recommend swapping 1. & 2. You can then use brew to install iterm2…
```brew cask instal iterm2```

Reply:
Thank you Jess.
1 & 2 now swapped